CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+ 3. The platform you are using (browser version and operating system).
+
+For documentation additions, you should first post an issue describing what you propose to add, where you'd like to add it in the documentation, and a description of why you think it's an important addition. For documentation improvements and fixes, you should post an issue describing what is currently wrong or missing, and how you propose to address it. For more information about building and contributing to Emperor's documentation, see [this guide](doc/README.md).
+
+When you post your issue, the Emperor developers will respond to let you know if we agree with the addition or change. It's very important that you go through this step to avoid wasting time working on a feature that we are not interested in including in Emperor.
+
+
+Getting started: "quick fixes"
+------------------------------
+
+Some of our issues are labeled as ``quick fix``. Working on [these issues](https://github.com/biocore/Emperor/issues?direction=desc&labels=quick+fix&milestone=&page=1&sort=updated&state=open) is a good way to get started with contributing to Emperor. These are usually small bugs or documentation errors that will only require one or a few lines of code to fix. Getting started by working on one of these issues will allow you to familiarize yourself with our development process before committing to a large amount of work (e.g., adding a new feature to Emperor). If you're interested in working on one of these issues, you should comment on the issue requesting that it be assigned to you.
+
+
+Code Review
+-----------
+
+When you submit code to Emperor, it will be reviewed by one or more Emperor developers. These reviews are intended to confirm a few points:
+
+* Your code is sufficiently well-tested (see Testing Guidelines below).
+* Your code adheres to our Coding Guidelines (see Coding Guidelines below).
+* Your code is sufficiently well-documented (see Coding Guidelines below).
+* Your code provides relevant changes or additions to Emperor (Type of Submissions above).
+
+This process is designed to ensure the quality of Emperor, and can be a very useful experience for new developers.
+
+Particularly for big changes, if you'd like feedback on your code in the form of a code review as you work, you should request help in the issue that you created and one of the Emperor developers will work with you to perform regular code reviews. This can greatly reduce development time (and frustration) so we highly recommend that new developers take advantage of this rather than submitting a pull request with a massive amount of code in one chunk. That can lead to frustration when the developer thinks they are done, but the reviewer requests large amounts of changes, and it is also very hard to review.
+
+
+Submitting code to Emperor
```

Comment:
This is a weird one, if you look at the source the numbering goes from
1-11, but it seems like GitHub is rendering this incorrectly. I've also
tried with Atom's markdown preview and Atom does render it from 1-11 so
I think this is a bug for GitHub.

On (Feb-22-15| 9:53), Antonio Gonzalez wrote:
> > +Code Review
> > +-----------
> > +
> > +When you submit code to Emperor, it will be reviewed by one or more Emperor developers. These reviews are intended to confirm a few points:
> > +
> > +* Your code is sufficiently well-tested (see Testing Guidelines below).
> > +* Your code adheres to our Coding Guidelines (see Coding Guidelines below).
> > +* Your code is sufficiently well-documented (see Coding Guidelines below).
> > +* Your code provides relevant changes or additions to Emperor (Type of Submissions above).
> > +
> > +This process is designed to ensure the quality of Emperor, and can be a very useful experience for new developers.
> > +
> > +Particularly for big changes, if you'd like feedback on your code in the form of a code review as you work, you should request help in the issue that you created and one of the Emperor developers will work with you to perform regular code reviews. This can greatly reduce development time (and frustration) so we highly recommend that new developers take advantage of this rather than submitting a pull request with a massive amount of code in one chunk. That can lead to frustration when the developer thinks they are done, but the reviewer requests large amounts of changes, and it is also very hard to review.
> > +
> > +
> > +Submitting code to Emperor
> 
> The numbering restarts after each code section, I do not think that should be the case. 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/348/files#r25134452